monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -169,6 +173,58 @@ def _check_data_uniformity(keys: list[str], result: dict) -> bool:
         return True
 
     def get_all_case_stats(self, key="training", transform_list=None):
+        """ Wrapper for the internal _get_all_case_stats to perform multi-gpu processing
+        """
+        if self.device.type == 'cpu':
+            nprocs = 1
+            print(f'Using CPU for data analyzing!')
```

Comment:
Thank you @heyufan1995 for preparing the PR.

Here we should use the logging instead of `print()`.